public/widget.js

```diff
@@ -0,0 +1,222 @@
+(function () {
+  const host = 'https://li.finance'
+  let config = {
+    elementId: 'lifi-widget',
+    mode: 'drawer', // default | drawer,
+    buttonText: 'Li.Fi Swap',
+    options: {
+      fromChain: 'eth',
+      fromToken: null,
+      toChain: null,
+      toToken: null,
+      fromAmount: null
+    },
+  };
+  const drawerSettings = {
+    activeClass: 'is-active',
+    visibleClass: 'is-visible',
+    targetAttribute: 'data-drawer-target',
+    triggerAttribute: 'data-drawer-trigger',
+    closeAttribute: 'data-drawer-close',
```

Comment:
The drawer gets injected directly, and is not inside an iframe right? Could general class names like `is-active` clash with user classes?

Reply:
Nice catch! 💪  I'll fix this.